doc/user/ospf6d.rst

```diff
@@ -610,3 +610,260 @@ Larger example with policy and various options set:
+      Authentication trailer is enabled with manual key         ==> new info added
+        Packet drop Tx 0, Packet drop Rx 0
+
+
+OSPFv3 supports options in hello and database description packets hence
+the presence of authentication trailer needs to be stored in OSPFv3
+neighbor info. Since RFC specifies that we need to handled sequence number
+for every ospf6 packet type, sequence number recvd in authentication header
+from the neighbor is stored in neighbor to validate the packet.
+json output is also supported.
+
+.. code-block:: frr
+
+    frr# show ipv6 ospf6 neighbor 2.2.2.2 detail
+     Neighbor 2.2.2.2%ens192
+        Area 1 via interface ens192 (ifindex 3)
+        0 Pending LSAs for LSUpdate in Time 00:00:00 [thread off]
+        0 Pending LSAs for LSAck in Time 00:00:00 [thread off]
+        Authentication header present                           ==> new info added
+                             hello        DBDesc       LSReq        LSUpd        LSAck
+          Higher sequence no 0x0          0x0          0x0          0x0          0x0
+          Lower sequence no  0x242E       0x1DC4       0x1DC3       0x23CC       0x1DDA
+
+Sent packet sequence number is maintained per ospf6 router for every packet
+that is sent out of router, so sequence number is maintained per ospf6 process.
+
+.. code-block:: frr
+
+    frr# show ipv6 ospf6
+     OSPFv3 Routing Process (0) with Router-ID 2.2.2.2
+     Number of areas in this router is 1
+     Authentication Sequence number info
+      Higher sequence no 3, Lower sequence no 1656
+
+Debug command:
+==============
+Below command can be used to enable ospfv3 authentication trailer
+specific logs if you have to debug the feature.
+
+.. clicmd:: debug ospf6 authentication [<tx|rx>]
+
+Feature supports authentication trailer tx/rx drop counters for debugging,
+which can be used to see if packets are getting dropped due to error in
+processing authentication trailer information in OSPFv3 packet.
+json output is also supported.
+
+.. code-block:: frr
+
+    frr# show ipv6 ospf6 interface ens192
+    ens192 is up, type BROADCAST
+      Interface ID: 5
+      Number of I/F scoped LSAs is 2
+        0 Pending LSAs for LSUpdate in Time 00:00:00 [thread off]
+        0 Pending LSAs for LSAck in Time 00:00:00 [thread off]
+      Authentication trailer is enabled with manual key
+        Packet drop Tx 0, Packet drop Rx 0                      ==> new counters
+
+Clear command:
+==============
+Below command can be used to clear the tx/rx drop counters in interface.
+Below command can be used to clear all ospfv3 interface or specific
+interface by specifying the interface name.
+
+.. clicmd:: clear ipv6 ospf6 auth-counters interface [IFNAME]
+
+
+Configuration Limits
+====================
+
+Ospf6d currently supports 100 interfaces addresses if MTU is set to
+default value, and 200 interface addresses if MTU is set to jumbo
+packet size or larger.
+
```

Comment:
This is not true and should be deleted.

Reply:
I am not sure how this got added to my changes. This is not part of my changes. Will remove it.